apps/native-component-list/package.json

```diff
@@ -139,19 +137,16 @@
     "moment": "^2.29.4",
     "path": "^0.12.7",
     "pixi.js": "^4.6.1",
-    "processing-js": "^1.6.6",
```

Comment:
We need to revert this packge, it's a peer dependency of `expo-processing` and without it NCL app build fails.